cpp/ps0002/main (4).cpp

```diff
@@ -1,29 +1,18 @@
-/******************************************************************************
-
-Welcome to GDB Online.
-GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
-C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
-Code, Compile, Run and Debug online from anywhere in world.
-
-*******************************************************************************/
 #include <iostream>
-
+using namespace std;
 int main()
 {
-	std::cout<<"Hello World";
-
-#include <iostream>
-	using namespace std;
-	int main()
-	{
-		int population_of_the_building =40;
-		int Alternative_Medicine = 4;
-		int Team_Mumbers_of_the_Programming_Company = 100;
-		float Price_of_a_loaf_of_bread = 5.5;
-		char The_Frist_Letter_of_a_persons_Name = 'S';
-		int Number_of_births = 134568;
-		float result = 3.2;
-		double deletectricity_bill_in_Egypt = 566899864212.456;
-		float Age = 245;
-		return 0;
-	}
\ No newline at end of file
+	int population_of_the_building = 40; // 0.5/0.75
+	int Alternative_Medicine = 4; // 0.75/0.75
+	int Team_Mumbers_of_the_Programming_Company = 100; // 0.75/0.75
+	float Price_of_a_loaf_of_bread = 5.5; // 0.75/0.75
+	char The_Frist_Letter_of_a_persons_Name = 'S'; // 0.75/0.75
+	int Number_of_births = 134568; // 0.75/0.75
+	double result = 3.2; // 0.5/0.75
+	double deletectricity_bill_in_Egypt = 566899864212.45; // 0.5/0.75
+	unsigned int Age = 2346445; // 0.5/0.75
+	// total 5.75/6.75
```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables